client/a8c-for-agencies/sections/purchases/payment-methods/payment-method-add/index.tsx

```diff
@@ -56,7 +61,17 @@ export default function PaymentMethodAdd( { withAssignLicense }: Props ) {
 				</LayoutHeader>
 			</LayoutTop>
 
-			<LayoutBody>Payment Method Add</LayoutBody>
+			<LayoutBody>
+				<div className="payment-method-add__content">
+					<Card className="payment-method-add__card payment-form">
+						<PaymentMethodForm />
+					</Card>
+
+					<Card className="payment-method-add__card aside">
+						<PaymentMethodStripeInfo />
```

Comment:
The Stripe info is purely presentational, so I decided not to port it. If the A4A deviates from the original design, we can always create a new one.